acl.go

```diff
@@ -162,8 +164,15 @@ func (odbi *ovndb) aclAddImp(lsw, direct, match, action string, priority int, ex
 
 	row["action"] = action
 	row["log"] = logflag
-	if logflag && len(meter) > 0 {
-		row["meter"] = meter
+	if logflag {
+		ok := odbi.meterFind(meter)
+		if ok {
+			row["meter"] = meter
+		}
+		switch severity {
+		case "alert", "debug", "info", "notice", "warning":
```

Comment:
why removing the default error case? If severity is invalid, we should error out instead of omit. Or if we want to let the server to validate, we should not do switch case, but simply assign the value. Current code will result in an invalid value being treated as "info" - the default.